test/attributes.test.ts

```diff
@@ -120,7 +120,7 @@ describe("Attributes", () => {
             }
         });
         expect(successionResponse.isSuccess).toBe(true);
-        const notificationResponse = await client1.attributes.notifyPeerAboutIdentityAttributeSuccession(successionResponse.result.successor.id, {
+        const notificationResponse = await client1.attributes.notifyPeerAboutRepositoryAttributeSuccession(successionResponse.result.successor.id, {
```

Comment:
Kindly, leave the name as it was: The notification sent to the peer contains an update to the _own shared identity attribute_, not a repository attribute. (I know .. there's room for three hours of bikeshedding about this particular name. Been there, done that and afterwards we named the runtime use case `notifyPeerAboutIdentityAttributeSuccession`.)

Reply:
since this concerns a public sdk method, this change would also have to apply to some types and the openAPI specification. We discussed those with a different conclusion yesterday, are you fine with those changing as well?

Reply:
I'm not sure I understand. I meant for you to leave the method's name as `notifyPeerAboutIdentityAttributeSuccession`. Just revert your changes for this particular method name and the names of any related files.

Please elaborate if I missed your point: who discussed what with what conclusion and what changes are you talking about?

Reply:
`notifyPeerAboutIdentityAttributeSuccession` is not correct imo. In the runtime the UseCase and facade methods are called `notifyPeerAboutRepositoryAttributeSuccession` so this should map this method.

Reply:
>In the runtime the UseCase and facade methods are called `notifyPeerAboutRepositoryAttributeSuccession`

That change went unnoticed to me. In that case we obviously apply the same naming here. Thus, forget what I said.